plugins/awards/src/components/AwardEditComponent/AwardEditComponent.tsx

```diff
@@ -232,10 +217,10 @@ export const AwardEditCard = ({ award = emptyAward }: AwardEditCardProps) => {
           />
           <Grid container alignItems="center">
             <Grid item>
-              <InputLabel>Award logo (150x50 px)</InputLabel>
+              <InputLabel>Award logo (3:1 aspect ratio)</InputLabel>
             </Grid>
             <Grid item>
-              <img alt="" src={awardImage} height="50" width="150" />
+              <img alt="" src={awardImage} height="200" width="600" />
```

Comment:
just picking an arbitrary “nice size”.

Reply:
@zhammer maybe we should have a big and a small image? The big one would be the displayed in the awards page itself, the small one in the badge.

Reply:
yep i was thinking we could add image resizing, though that may be a good amount of work so was going to leave that for the future. for now showing the same image in both places shouldn't be too much trouble